app/src/main/java/org/wikipedia/page/PageActivity.java

```diff
@@ -452,12 +447,8 @@ public void loadPage(@NonNull final PageTitle title,
         });
     }
 
-    /**
-     * Go directly to the Main Page of the current Wiki, optionally allowing state loss of the
-     * fragment manager. Useful for when this function is called from an AsyncTask result.
-     */
-    public void loadMainPage(TabPosition position) {
-        PageTitle title = MainPageClient.getMainPageTitle();
+    public void loadEmptyPage(TabPosition position) {
+        PageTitle title = new PageTitle(Constants.EMPTY_PAGE_TITLE, app.getWikiSite());
         HistoryEntry historyEntry = new HistoryEntry(title, HistoryEntry.SOURCE_MAIN_PAGE);
```

Comment:
We should probably have an empty source?

Reply:
I changed the source to `SOURCE_INTERNAL_LINK` so that it's slightly more consistent with what this is.  I don't know if we need a new constant for an "empty" source, since it won't be used anywhere else except here, and no other part of the app will care about the source of this entry.